pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,134 @@
+{ config
+, pkgs
```

Comment:
Right, I think this was leftover from when I was using it in my personal config and I forgot to change it. Same with disabling tests, i'll remove both.